RelayProofsOfConcept/EddsaTendermintServer/tools/docker/start-tendermint-local-testnet.sh

```diff
@@ -0,0 +1,33 @@
+CURDIR=$(pwd)
+# echo "$CURDIR"
+
+# build white-city binaries 
+cargo build --all --release
+
+# build white-city server image
+docker-compose build
+
+# if empty block
+./tools/docker/gen-empty-block-cluster.py $1 $2 $3> ./tools/docker/docker-compose.yml
+
+# if nonempty block
+# ./tools/docker/gen-nonempty-block-cluster.py $1 $2 $3> ./tools/docker/docker-compose.yml
+
+cp ./Dockerfiles/tendermint/localnode $GOPATH/src/github.com/tendermint/tendermint/networks/local/localnode/Dockerfile
+cp $GOPATH/src/github.com/tendermint/tendermint/Makefile $GOPATH/src/github.com/tendermint/tendermint/Makefile.bak
+cp $GOPATH/src/github.com/tendermint/tendermint/docker-compose.yml $GOPATH/src/github.com/tendermint/tendermint/docker-compose.yml.bak
+cp ./tools/docker/tendermint_Makefile $GOPATH/src/github.com/tendermint/tendermint/Makefile
+cp ./tools/docker/docker-compose.yml $GOPATH/src/github.com/tendermint/tendermint/docker-compose.yml
```

Comment:
Is it possible to avoid having to clone Tendermint and build it from source?

Reply:
yes 

because of the time limit, this is actually for the convenience of demo and testing

I will clean these up and make it a truely ready  PR in the future